ui/src/main/webapp/app/query/date-range/date-range.html

```diff
@@ -0,0 +1,25 @@
+<md-card>
+    <md-card-title>
+        <md-card-title-text>
+            <span class="md-headline">In this date range</span>
+        </md-card-title-text>
+    </md-card-title>
+    <ng-form name="ctrl.dateForm">
+        <div layout="row">
+            <md-input-container>
+                <label>Start Date</label>
+                <md-datepicker ng-model="ctrl.start" name="start" id="start-date" ng-change="ctrl.onStartDateUpdate()"></md-datepicker>
+                <div ng-messages="ctrl.dateForm.start.$error">
+                    <div ng-message="valid">The start date entered is not a date.</div>
+                </div>
+            </md-input-container>
+            <md-input-container>
+                <label>End Date</label>
```

Comment:
Might be worth adding a tool tip to tell users that these date ranges are inclusive.